### ISSUE TITLE
Quantizing last layer for output prediction/logits  

### ISSUE THREAD

**Issue by DeepCowProductions** [fhswf/eki-transformer-dev#27]
Currently, we can not quantize this layer, otherwise our model does not converge properly.
When training from scratch, it does not converge at all when all other layers are also quantized.
